packages/lib/services/plugins/RepositoryApi.ts

```diff
@@ -79,21 +81,45 @@ export default class RepositoryApi {
 
 		await this.loadManifests();
 		await this.loadRelease();
+		await this.loadStats();
 	}
 
 	private async loadManifests() {
 		const manifestsText = await this.fetchText('manifests.json');
 		try {
 			const manifests = JSON.parse(manifestsText);
 			if (!manifests) throw new Error('Invalid or missing JSON');
-
 			this.manifests_ = Object.keys(manifests).map(id => {
 				const m: PluginManifest = manifests[id];
+
 				// If we don't control the repository, we can't recommend
 				// anything on it since it could have been modified.
 				if (!this.isUsingDefaultContentUrl) m._recommended = false;
 				return m;
 			});
+			// await this.loadStats();
+		} catch (error) {
+			throw new Error(`Could not parse JSON: ${error.message}`);
+		}
+	}
+
+	public async loadStats() {
+		try {
+			const statsText = await this.fetchText('stats.json');
+			const stats = JSON.parse(statsText);
+			if (!stats) throw new Error('Invalid or missing JSON');
+			let tempDownloadCount: number = 0;
+			let created_date = '';
+			this.manifests_.forEach(manifest => {
+				stats[manifest.id] && Object.entries(stats[manifest.id]).forEach((stats: any[], index) => {
+					tempDownloadCount += stats[1].downloadCount;
+					if (index === 0) created_date = stats[1].createdAt;
+				});
+				const manifestObj = this.manifests_.find(obj => obj.id === manifest.id);
+				manifestObj._totalDownloads = tempDownloadCount;
+				manifestObj._created_date = created_date;
```

Comment:
Why the random mix of camelCase and snake_case? It should be snake_case based on the existing manifest definition. created_date should be a Date object